docs/operator/troubleshooting.md

```diff
@@ -0,0 +1,88 @@
+# Troubleshooting the Starboard Operator
+
+The Starboard operator installs several Kubernetes resources into the `starboard-system` namespace.
```

Comment:
Based on our installation docs we demonstrate how to install in in `starboard-system` namespaces. But in practice, you can install it in any namespace. Do you think we can clarify that as well?